packages/ethereum-contracts/contracts/utils/SuperfluidFrameworkDeploymentSteps.sol

```diff
@@ -347,9 +349,17 @@ library SuperfluidGovDeployerLibrary {
     }
 }
 
+library SuperfluidDMZForwarderDeployerLibrary {
+    function deploy() external returns (DMZForwarder) {
```

Comment:
Super nitpick but I guess this could use a small comment to not forget the transferring of ownership.